### PR TITLE
[Cleanup] Remove nil-default completion parameters

### PR DESCRIPTION
**Pull request by BasThomas**

### CODE REVIEW

File: Classes/Image Upload/ImageUploadTableViewController.swift [hunk 1]
```diff
@@ -189,14 +189,12 @@ class ImageUploadTableViewController: UITableViewController {
 
                 switch result {
                 case .error:
-                    print("error")
                     ToastManager.showGenericError()
                     self?.setRightBarItemIdle()
 
                 case .success(let link):
-                    print("success")
```
Comment (BasThomas) [inline Classes/Image Upload/ImageUploadTableViewController.swift]: Also removed these prints, I felt like if there was a good reason to keep them they should at least be more descriptive. (cc @Sherlouk)

Comment (Sherlouk) [reply Classes/Image Upload/ImageUploadTableViewController.swift]: Debug messages, forgot to remove!

Comment (BasThomas) [reply Classes/Image Upload/ImageUploadTableViewController.swift]: That's what I thought, cool!

File: Classes/Repository/RepositoryIssuesViewController.swift [hunk 2]
```diff
@@ -84,7 +84,7 @@ SearchBarSectionControllerDelegate {
     // MARK: SearchBarSectionControllerDelegate
 
     func didChangeSelection(sectionController: SearchBarSectionController, query: String) {
-        filter(query: query.trimmingCharacters(in: .whitespacesAndNewlines), animated: true)
+        filter(query: query, animated: true)
```
Comment (BasThomas) [inline Classes/Repository/RepositoryIssuesViewController.swift]: Removed as this is redundant; it already happens in [`filtered()`](https://github.com/rnystrom/GitHawk/blob/master/Classes/Utility/Filterable.swift#L17).

**Pull request merged**